agir/api/settings.py

```diff
@@ -752,7 +752,9 @@
 
 # nuntius
 NUNTIUS_REDIS_CONNECTION_GETTER = "agir.api.redis.get_auth_redis_client"
-NUNTIUS_PUBLIC_URL = FRONT_DOMAIN
+NUNTIUS_PUBLIC_URL = os.environ.get(
+    "NUNTIUS_PUBLIC_URL", "https://www.actionpopulaire.fr"
+)
```

Comment:
Ce n'est pas mieux de garder la variable d'environnement comme valeur par défaut ?
```suggestion
NUNTIUS_PUBLIC_URL = os.environ.get(
    "NUNTIUS_PUBLIC_URL", FRONT_DOMAIN
)
```